packages/wonder-blocks-dropdown/components/action-menu.js

```diff
@@ -109,69 +126,153 @@ export default class ActionMenu extends React.Component<Props, State> {
         };
     }
 
-    /**
-     * Update the internal state and notify the parent component when menu is
-     * toggled
-     */
-    handleToggleMenu = (opened: boolean) => {
+    handleItemSelected = () => {
+        // close menu
+        this.handleOpenChanged(false);
+
+        // Bring focus back to the opener element.
+        if (this.openerElement) {
+            this.openerElement.focus();
+        }
+    };
+
+    handleOpenChanged = (opened: boolean, keyboard?: boolean) => {
         this.setState({
-            opened: opened,
+            opened,
+            keyboard,
         });
 
         if (this.props.onToggle) {
             this.props.onToggle(opened);
         }
     };
 
+    handleOptionSelected = (selectedValue: string) => {
```

Comment:
It looks like most of the changes in this file were from copy/pasting from dropdown.js, is that correct?

Reply:
Yes, I'm trying to transfer the good parts from dropdown to action-menu. Then, I'll delete dropdown.js in an upcoming PR (different ticket, tho).